client/src/partials/templates/grid/credit_equiv.cell.html

```diff
@@ -1,5 +1,5 @@
 <div class="ui-grid-cell-contents" title="TOOLTIP" ng-if="!row.groupHeader">
-  {{ row.entity.credit_equiv | currency:grid.appScope.enterprise.currency_id }}
+  {{row.entity.credit_equiv}}
```

Comment:
I assume these are temporary fixes.  Would you mind creating an issue to change the currency filters to support longer numbers?